.circleci/generate-ci.py

```diff
@@ -10,6 +10,14 @@
 config = yaml.load(Path("config.pre.yml"))
 build_sink_deps = []
 
+# Insert an entry into config.gen.yml in the build workflow:
```

Comment:
You should update the Makefile to use this file rather than `make.sh` for doing local checking of the format

Reply:
I don't think make.sh is used anywhere

Reply:
I meant this https://github.com/reach-sh/reach-lang/blob/master/.circleci/Makefile#L8-L9 but you fixed it. Sorry!